src/ansys/aedt/toolkits/magnet_segmentation/backend/workflows/aedt.py

```diff
@@ -364,3 +364,14 @@ def _get_project_materials(self):
         self.aedtapp = None
 
         return mats
+
+    def _get_setups(self):
+        """Get design setups."""
+        self.connect_design(app_name=list(properties.active_design.keys())[0])
+
+        setup_names = [setup.name for setup in self.aedtapp.setups]
+
+        self.aedtapp.release_desktop(False, False)
+        self.aedtapp = None
```

Comment:
Quick question, do you need to release the desktop after retrieving the design setups ? Same for the aedtapp ?